apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-tabbed-example.component.ts

```diff
@@ -22,20 +23,22 @@ export class PlatformDynamicPageTabbedExampleComponent {
 
     pageTitle = 'Balenciaga Tripple S Trainers';
 
+    constructor(private _overflowHandlingService: PlatformDynamicPagePageOverflowService) {}
+
     onCollapseChange(event: DynamicPageCollapseChangeEvent): void {
         console.log('collapse changed');
     }
 
     openPage(): void {
         this.overlay.nativeElement.style.width = '100%';
         this.fullscreen = true;
-        document.getElementById('page-content').style.overflowY = 'hidden'; // hide the underlying page scrollbars
+        this._overflowHandlingService.isExampleOpened.next(true);
     }
     closePage(event: Event): void {
         event.stopPropagation();
         this.fullscreen = false;
         this.overlay.nativeElement.style.width = '0%';
-        document.getElementById('page-content').style.overflowY = 'auto';
+        this._overflowHandlingService.isExampleOpened.next(false);
```

Comment:
@kavya-b BTW since these examples should run on stackblitz this solution doesn't reveal internal structure of the doc app (`pageContentEl.style.overflowY` would fail on `stackblitz`)
The only thing we have to register `PageOverflowService` file to make it accessible on `stackblitz`